modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -235,6 +235,21 @@ export default function DatePicker({
 		}
 	};
 
+	const [expanded, setExpanded] = useState(false);
+
+	const handleExpandedChange = (value) => {
+		if (value !== expanded) {
```

Comment:
@marcosapmf you did that to avoid call onFocus or onBlur twice in a row, right?
@matuzalemsteles isn't that a bug on `onExpandedChange` that supposed to be called only once by state change?

Reply:
I see, we probably have a bug here, apparently, the method is called more than once internally with the click out. I'll link an issue on Clay to fix, thanks for figuring this out.

Reply:
I open an issue for this https://github.com/liferay/clay/issues/4928